scripts/postgenerate.mjs

```diff
@@ -0,0 +1,12 @@
+import { cssTokenResolve } from './cssTokenResolve.mjs';
+import { generateTailwindTokens } from './tokenToTailwindParser.mjs';
+import { splitComponentTokens } from './splitComponentTokens.mjs';
+
+try {
+  const resolvedFilesPath = 'css-resolved';
```

Comment:
@mercury83 @RudolfDaniel @pvidoczi If you can come up with a better name for the folder of resolved CSS files, it would be much appreciated